.github/workflows/ci.yml

```diff
@@ -2,44 +2,123 @@ name: Garage CI
 
 env:
   GARAGE_GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  TRAVIS_COMMIT_RANGE: origin/${{ github.base_ref }}...origin/${{ github.head_ref }}
+  COMMIT_RANGE: origin/${{ github.base_ref }}...origin/${{ github.head_ref }}
   DOCKER_TAG: garage-ci-${{ github.run_id }}
   OWNER: "rlworkgroup"
   DOCKER_CACHE_REPO: "garage-ci"
+  MJKEY: ${{ secrets.MJKEY }}
 
 on:
+  schedule:
+    - cron: '0 9 * * *'
+  push:
+    branches: [master]
```

Comment:
Can you make this consistent with line 17?

Reply:
is the dash unnecessary? In the github actions documentation, it has it for cron but not branches.

Reply:
Oh no. I meant make both of them either `[ master ]` or `[master]`